src/client/src/models/Scene.js

```diff
@@ -41,41 +41,66 @@ const Scene = {
     return JSON.parse(localStorage.getItem('scene-list'))
   },
 
+  findSceneIndex: function(sceneid) {
+    // get current list
+    const list = JSON.parse(localStorage.getItem('scene-list'))
+
+    for (var i = 0; i < list.length; i++) {
+      if (sceneid == list[i].id) {
+        return i
+      }
+    }
+
+    return -1
+  },
+
   addScene: function() {
     // get current list
     const list = JSON.parse(localStorage.getItem('scene-list'))
+      // keeps tracks of id numbers in order to not crash into each other
+    var scenes_index = 0
+    // Finds the max scene id in the list
+    if (list.length == 0) {
+      scenes_index = 0
+    } else {
+      scenes_index = 0
+      for (var i = 0; i < list.length; i++) {
+        if (scenes_index < list[i].id) {
+          scenes_index = list[i].id
+        }
+      }
+    }
+    // Adds a new scene
+    scenes_index = scenes_index + 1
 
     // add new scene object
     list.push({
-      id: list.length + 1,
+      id: scenes_index,
```

Comment:
`scenes_index` is a little misleading here, if the intent is to add a single scene i think we should add a single scene_index

Reply:
Yup and fixed it to reflect and new name called `scene_max`. It finds the current max scene id and appends a new one to a new scene. It does it in order to not have scene id match, since every scene id is unique.